tests/Command/ExportListCommandsCommandTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Tests\Command;
```

Comment:
Once the command move in the proper namespace we will keep this in the App namespace because it's an integration test but we will move it to:

App\Tests\Console\Command;